Apps/Encounter/src/Models/EncounterHistory.cs

```diff
@@ -0,0 +1,101 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Encounter.Models
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text.Json.Serialization;
+
+    /// <summary>
+    /// Represents a patient Encounter.
+    /// </summary>
+    public class EncounterHistory
```

Comment:
It seems to me that we have issue with the naming for the models. Personally, I prefer to call this EncounterDTO  
or ClaimDTO instead so we don't have to think or come up with a new suffix for the new domain models.

Reply:
what does DTO stand for?

Reply:
it stands for Data Transfer Object which is generally used to expose the object to api client / client UI <ref. https://medium.com/@Colin_But/dissecting-the-dto-pattern-ac3e54d0e4c8>

Reply:
Not 100% sure is better, but I rather see something like EncounterData / EncounterModel.

Reply:
These sounds good to me to. Btw, just trying to prove this DTO suffix naming to see if you are ok.
https://github.com/PacktPublishing/ASP.NET-Core-3-and-Angular-9-Third-Edition/tree/ca0aca9df29be086bfe736c2c83cc15ed7f31c29/Chapter_07/WorldCities/Data is the source code from a C# MVP guys who wrote the good "ASP.NET Core 3 and Angular 9" book.
Check it out and see if we can decide to use ...DTO (...Dto) vs ...Data / ...Model

Reply:
Lets go with Model then. In general, I prefer not to use acronyms on variable or/and class names since it can bloat the meaning really fast

Reply:
Sounds good, I update my PR as well